File: app/src/main/java/org/michaelbel/moviemade/rest/model/Company.java
package org.michaelbel.moviemade.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import io.realm.RealmObject;

public class Company extends RealmObject implements Serializable {

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public String name;

    @SerializedName("description")
    public String description;

    @SerializedName("headquarters")
    public String headquarters;

    @SerializedName("logo_path")
    public String logoPath;

    @SerializedName("parent_company")
    public String parentCompany;
}

File: app/src/main/java/org/michaelbel/moviemade/data/dao/Company.kt
package org.michaelbel.moviemade.data.dao

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Company(
    @SerializedName("description") val description: String,
    @SerializedName("headquarters") val headquarters: String,
    @SerializedName("homepage") val homepage: String,
    @SerializedName("id") val id: Int,
    @SerializedName("logo_path") val logoPath: String,
    @SerializedName("name") val name: String,
    @SerializedName("origin_country") val originCountry: String,
    @SerializedName("parent_company") val parentCompany: Company
) : Serializable